packages/Webkul/Velocity/src/Resources/views/shop/layouts/master.blade.php

```diff
@@ -59,11 +59,11 @@
 
                             {{-- mobile header --}}
                             <div class="vc-small-screen container">
-                                @include('velocity::shop.layouts.header.mobile')
+                                @include('shop::layouts.header.mobile')
```

Comment:
This is only for the velocity theme. Because changes are not made for the default theme.

This will give an error if the user switches to the default theme.

Reply:
@devansh-webkul 
As we discuss, this is not an issue, So i am closing this PR.